components/renku_data_services/authz/authz.py

```diff
@@ -620,6 +616,10 @@ async def _remove_project(
         zed_token: ZedToken | None = None,
     ) -> _AuthzChange:
         """Remove the relationships associated with the project."""
+        if not project.id:
```

Comment:
A bit confused by this (and the other occurrences below), this can't be None with the changes I did?

Reply:
I started the PR from before the change you made was merged. Now they are not needed. I removed them.